pkg/controllers/cassandra/service/control_test.go

```diff
@@ -55,4 +55,16 @@ func TestServiceSync(t *testing.T) {
 			}
 		},
 	)
+	t.Run(
+		"service with outside owner",
+		func(t *testing.T) {
+			f := casstesting.NewFixture(t)
+			// Remove the ports from the default cluster and expect them to be
+			// re-created.
```

Comment:
This comment seems to be a c+p error?

Reply:
Happy to leave this here though and you can remove it in the upstream branch or something if you're going to be working in this file again anyway 😄  your call

Reply:
Ah yes. I'd prefer to fix that in a followup branch rather than wait for another round of CI.